lib/core/src/Cardano/Wallet/Shelley/Launch/Cluster.hs

```diff
@@ -1049,49 +1049,49 @@ withCluster tr dir LocalClusterConfig{..} faucetFunds onClusterStart = bracketTr
             (if postAlonzo then addGenesisPools else federalizeNetwork)
 
         if postAlonzo
-        then do
-            port0:ports <- rotate <$> randomUnusedTCPPorts nPools
-            let pool0:otherPools = configuredPools
-
-            let pool0Cfg = NodeParams
-                    genesisFiles
-                    cfgLastHardFork
-                    port0
-                    cfgNodeLogging
-            operatePool pool0 pool0Cfg $ \runningPool0 -> do
-                extraClusterSetupUsingNode configuredPools runningPool0
-                launchPools
-                    otherPools
-                    genesisFiles
-                    ports
-                    runningPool0
-                    onClusterStart
-        else do
-            -- NOTE: We should soon be able to drop Alonzo support here after
-            -- the Vasil HF, which should enable some simplifications of the
-            -- logic in 'withCluster'.
-            ports <- rotate <$> randomUnusedTCPPorts (1 + nPools)
-            let bftCfg = NodeParams
-                    genesisFiles
-                    cfgLastHardFork
-                    (head ports)
-                    cfgNodeLogging
-            withBFTNode tr dir bftCfg $ \runningBFTNode -> do
-                extraClusterSetupUsingNode configuredPools runningBFTNode
-
-                -- NOTE: We used to perform 'registerViaTx' as part of 'launchPools'
-                -- where we waited for the pools to become active (e.g. be in
-                -- the stake distribution) in parallel. Just submitting the
-                -- registration certs in sequence /seems/ to work though, and the
-                -- setup working 100% correctly in alonzo will soon not be
-                -- important.
-                mapM_ (`registerViaTx` runningBFTNode) configuredPools
-                launchPools
-                    configuredPools
-                    genesisFiles
-                    (tail ports)
-                    runningBFTNode
-                    onClusterStart
+            then do
```

Comment:
The previous indentation was intentional to respect the 80-char line length. Why change it?

Reply:
I was fixing compilation errors similar to this one:
```
src/Cardano/Wallet/Shelley/Launch/Cluster.hs:1176:12: error:
    Unexpected semi-colons in conditional:
        if length (filter isRight group) /= poolCount; then do cancelAll
                                                               let errors
                                                                     = show (filter isLeft group)
                                                               throwIO
                                                                 $ ProcessHasExited
                                                                     ("cluster didn't start correctly: "
                                                                        <> errors)
                                                                     (ExitFailure
                                                                        1); else do let node = ...
                                                                                    action node
                                                                                      `finally`
                                                                                        cancelAll
    Perhaps you meant to use DoAndIfThenElse?
     |
1176 |         if length (filter isRight group) /= poolCount
     |     
```
... had to indent `then/else` relative to `if`.
Also, I made sure that 80 chars is not violated.

Reply:
I tried de-indenting `then/else` here, and `cabal test cardano-wallet:integration` compiled fine. Are you sure you get these errors 🤔